portal/server/controllers/dashboard/deals/template-filters.ff-test.js

```diff
@@ -5,7 +5,7 @@ import CONTENT_STRINGS from '../../../content-strings';
 import { generateFiltersArray } from '../filters/generate-template-filters';
 
 describe('controllers/dashboard/deals - template-filters', () => {
-  describe('deal status filters - FF_TFM_FACILITY_END_DATE_ENABLED enabled', () => {
+  describe('deal status filters - FF_TFM_DEAL_CANCELLATION_ENABLED enabled', () => {
```

Comment:
should the tests be any different for `FF_TFM_DEAL_CANCELLATION_ENABLED ?`

Reply:
This was a typo in the original tests - they dont depend at all on `FF_TFM_FACILITY_END_DATE_ENABLED`

Reply:
Should the title or test be changed if we are removing the FF.